### PR TITLE
Media library settings api update

### PR DESCRIPTION
**Pull request by soupette**
#### Description of what you did:
This PR improves the current settings API by allowing a plugin to add a link into the global section.
Needs to be merged after #5264 

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [ ] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [ ] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [ ] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: docs/3.0.0-beta.x/plugin-development/frontend-settings-api.md [hunk 1]
```diff
@@ -191,3 +191,85 @@ export default strapi => {
...
+    preventComponentRendering: false,
+    settings: {
+      // Add a link into the global section of the settings view
+      global: [
+        {
+          title: 'Setting link 1',
+          to: `${strapi.settingsBaseURL}/setting-link-1`,
+          name: 'settingLink1',
+          Component: SettingLink,
+          // True or false
```
Comment (virginieky) [inline docs/3.0.0-beta.x/plugin-development/frontend-settings-api.md]: Is this comment still relevant?

File: packages/strapi-admin/admin/src/containers/SettingsPage/index.js [hunk 2]
```diff
@@ -6,29 +6,32 @@
...
+    .map(({ to, Component, exact }) => (
+      <Route path={to} key={to} component={Component} exact={exact || false} />
+    ))
+    .filter((route, index, refArray) => {
+      return refArray.findIndex(obj => obj.key === route.key) === index;
+    });
 
-    return acc;
-  }, []);
+  console.log(createdRoutes);
```
Comment (virginieky) [inline packages/strapi-admin/admin/src/containers/SettingsPage/index.js]: Remove log

File: docs/3.0.0-beta.x/plugin-development/frontend-settings-api.md (hunk 1 above)
Comment (soupette) [reply docs/3.0.0-beta.x/plugin-development/frontend-settings-api.md]: Yes

File: packages/strapi-admin/admin/src/containers/SettingsPage/index.js (hunk 2 above)
Comment (HichamELBSI) [inline packages/strapi-admin/admin/src/containers/SettingsPage/index.js]: console.log

**Pull request merged**